reana_commons/serial.py

```diff
@@ -58,6 +59,12 @@
                         "type": "integer",
                         "default": "None",
                     },
+                    "kubernetes_memory_limit": {
```

Comment:
It might be good to enrich [yadage-schemas](https://github.com/yadage/yadage-schemas/blob/260fbc40ec5b8afa7d59c0dd00591e02e13774e4/yadageschemas/packtivity/environment/docker-enc-schema.json#L21) to enforce that the pattern is followed.

Something like this (untested).

```json
"resources": {
  "type": "array",
  "title": "Resources",
  "default":[],
  "items": {
    "type": "object",
    "properties": {
      "kubernetes_memory_limit": {
        "type": "string",
        "default": "",
        "title": "Kubernetes memory job limit",
        "pattern": "\d+[EPTGMK]i?\b",
     }
  }
},
```

WDYT?

Reply:
@mvidalgarcia Maybe we can add this pattern + ` "additionalProperties": false` together in `yadage-schemas`?

Can you create a issue in `yadage-schemas` and we can create a PR for it later. Thanks

Reply:
Perhaps this could be something also useful for yadage CC @lukasheinrich ?